solr/solr-ref-guide/modules/deployment-guide/pages/solrcloud-distributed-requests.adoc

```diff
@@ -184,22 +184,26 @@ One or more replica types that are preferred.
 Any combination of `PULL`, `TLOG` and `NRT` is allowed.
 
 `replica.location`::
-One or more replica locations that are preferred.
-+
-A location starts with `http://hostname:port`.
-Matching is done for the given string as a prefix, so it's possible to e.g., leave out the port.
-+
-A special value `local` may be used to denote any local replica running on the same Solr instance as the one handling the query.
+Prefer replicas that match the given location. Available options are:
+- `local` - Replicas in the same Solr instance as the one handling the query.
 This is useful when a query requests many fields or large fields to be returned per document because it avoids moving large amounts of data over the network when it is available locally.
 In addition, this feature can be useful for minimizing the impact of a problematic replica with degraded performance, as it reduces the likelihood that the degraded replica will be hit by other healthy replicas.
 +
 The value of `replica.location:local` diminishes as the number of shards (that have no locally-available replicas) in a collection increases because the query controller will have to direct the query to non-local replicas for most of the shards.
 +
 In other words, this feature is mostly useful for optimizing queries directed towards collections with a small number of shards and many replicas.
+- `host` - Replicas in Solr instances running on the same host as the one handling the query.
```

Comment:
[+1] You revised these at some point and this all comes off much more clearly IMO.  Good stuff 🎉